tests/catch_up_subscription_handles_errors.php

```diff
@@ -97,7 +97,11 @@ function (
                 ResolvedEvent $resolvedEvent
             ) use (&$props1): Promise {
                 $props1['raisedEvents'][] = $resolvedEvent;
-                $props1['raisedEventEvent']->resolve(true);
+                try {
+                    $props1['raisedEventEvent']->resolve(true);
+                } catch (Throwable $e) {
+                    // on connection close this event may appear twice, just ignore second occurrence
```

Comment:
shouldn't we make sure it doesn't happen twice? it doesn't feel intended to me

Reply:
When the connection gets cut off, you may receive the message twice. That's intended. The test only covers, that the message arrives at least once, which is does.

Reply:
okay